framework/Optimizers/GradientBasedOptimizer.py

```diff
@@ -265,86 +267,124 @@ def checkConvergence(self):
         break
     return convergence
 
-  def _updateConvergenceVector(self, traj, varsUpdate, currentLossValue):
+  def _updateConvergenceVector(self, traj, varsUpdate, currentLossVal):
     """
       Local method to update convergence vector.
       @ In, traj, int, identifier of the trajector to update
       @ In, varsUpdate, int, current variables update iteration number
-      @ In, currentLossValue, float, current loss function value
+      @ In, currentLossVal, float, current loss function value
```

Comment:
This was just to be self-consistent with other variable names, like "oldVal".  It was less work to change this one than the other ones, if I recall correctly.